package.json

```diff
@@ -30,11 +30,11 @@
   },
   "homepage": "https://github.com/avenga/eslint-config#readme",
   "peerDependencies": {
-    "@typescript-eslint/eslint-plugin": "~2.28.0",
-    "@typescript-eslint/parser": "~2.28.0",
-    "eslint": ">=6.8",
-    "eslint-plugin-import": "~2.20.2",
-    "eslint-plugin-prefer-arrow": "~1.2.1"
+    "@typescript-eslint/eslint-plugin": "^5.0.0",
+    "@typescript-eslint/parser": "^5.0.0",
+    "eslint": "^6.0.0 || ^7.0.0 || ^8.0.0",
```

Comment:
Die `^6.0.0` Angabe gestattet hier ein Downgrade, die Version 6.8 hat doch sicher einen Grund, warum sie gewählt wurde. Die ausgewählte `@typescript-eslint/eslint-plugin` Version benötigt z.B. aber auch schon `eslint@^8.15.0` daher halte ich die anderen Angaben hier auch für fragwürdig.

Reply:
changed accoring to https://typescript-eslint.io/users/dependency-versions